### ISSUE TITLE
Feature request: move vanilla AI into a different slot

### ISSUE THREAD

**Issue by gynt** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#19]
**Is your feature request related to a problem? Please describe.**
Some users want to compare Rats to each other for example.

**Describe the solution you'd like**
To accommodate this, it is interesting if the GUI would allow vanilla AI to be placed into a different slot.

**Describe alternatives you've considered**
Moving vanilla retraced things into those slots, but the portrait, lines, and speech is not provided so it isn't the same thing.

**Additional context**
We could ask FireFly if such an extension is allowed, i.e., if we are allowed to redistribute those files.

**Comment by Heroesflorian** [comment]
Could it work to gather the vanilla AIs' files from the user's pre-existing vanilla installation while installing the UCP, basically packaging them into a UCP3 AI mod package locally? 
The mappings are known, and that way there would not have to be any redistribution of those files via the UCP or some online package. 

Side note: That same functionality would also essentially double as an automated conversion/packaging tool for old AI mods to the UCP3 format, when applied on a non-vanilla installation that contains modded AIs.

**Comment by gynt** [comment]
True! Didn't think about that use case too... Let's give this some more priority
